mysqlebs.py

```diff
@@ -543,6 +585,47 @@ def os_fs_unfreeze(self, mountpoints):
 
         return True
 
+    def monitor_snapshot(self, responses):
+        """ Monitor snapshot requests to ec2.
+        """
+
+        if self.opts.dryrun:
+            return True
+
+        snapshotIDs = []
+
+        for resp in responses:
+            snapshotIDs.append(resp.get("SnapshotId"))
```

Comment:
evan and i worked this out in dms, if at and point this causes an issue it will be caught here


```
        if len(snapshots) != len(responses):
            #Trigger a warning, perhaps critical
            self.logger.debug('Attempted snapshot length does not match returned snapshot describe query length')
```